addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -119,8 +119,7 @@
                     <IdPaese t-esc="buyer.country_id.code"/>
                     <IdCodice t-esc="'0000000'"/>
                 </IdFiscaleIVA>
-                <CodiceFiscale t-if="not buyer.vat and buyer.l10n_it_codice_fiscale" t-esc="normalize_codice_fiscale(buyer.l10n_it_codice_fiscale)"/>
-                <CodiceFiscale t-elif="not buyer.vat and not buyer.l10n_it_codice_fiscale" t-esc="99999999999"/>
```

Comment:
@lordkrandel Why do you remove this L123?  It is still needed...

Reply:
No, it's no longer needed; Tony is right asking for removal:
![image](https://user-images.githubusercontent.com/1665365/212057667-d026172a-982c-4785-b6e6-0c4124f1601b.png)

Case 1) Country IT, no VAT no CodiceFiscale, rejected by checks
![image](https://user-images.githubusercontent.com/1665365/212057764-efb2ae76-4bca-4990-ac80-1ee4d8d38713.png)
Case 2) Country not-IT, no VAT no CodiceFiscale, a default `0000000` VAT is put.
![image](https://user-images.githubusercontent.com/1665365/212057871-d7a8b432-7bce-4db1-a0a0-a6d44952262c.png)

The PR https://github.com/odoo/odoo/pull/37346 that introduced that fake 99999999... Codice Fiscale mentioned the SDI checks where one of VAT number or CodiceFiscale was needed for a check, and it was a problem for non-Italian clients.
So, we should be covered.
![image](https://user-images.githubusercontent.com/1665365/212058735-1e1a05de-f113-45d6-938d-c7abf08c0163.png)

Reply:
@lordkrandel Thanks for the explanation!